cypress/support/pages/homeСatalogue.pageObject.js

```diff
@@ -17,6 +17,17 @@ class HomeAndCataloguePageObject extends PageObject {
     cy.contains('.hrefch', product)
       .click();
   }
+
+  addToCartBtn() {
+    cy.contains('.btn', 'Add to cart')
+      .click();
+  }
+
+  placeOrderBtn() {
```

Comment:
Function name 'placeOrderBtn' should be a verb or verb phrase. Consider renaming it to 'clickPlaceOrderBtn' to better reflect its action.